nano/node/bootstrap/bootstrap_ascending.hpp

```diff
@@ -68,12 +69,13 @@ namespace bootstrap
 		class account_sets
 		{
 		public:
-			explicit account_sets (nano::stat &);
+			explicit account_sets (nano::stat &, nano::store & store);
 
 			/**
 			 * If an account is not blocked, then add it to the forwarding set and, additioally, if it does not exist in the backoff set, set the priority.
 			 */
-			void prioritize (nano::account const & account, float priority);
+			void priority_up (nano::account const & account);
```

Comment:
It would be nice to comment this block of functions:
```
void priority_up (nano::account const & account);
void priority_down (nano::account const & account);
void block (nano::account const & account, nano::block_hash const & dependency);
void unblock (nano::account const & account, nano::block_hash const & hash);
void force_unblock 
 ```